dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Content/Site.css

```diff
@@ -193,6 +196,7 @@ td.ms-row {
 }
 .ms-thumbnail img{
     height: 50px;
+    width: 50px;
```

Comment:
The comments of the API specify that they should be square and the suggested size. The problem I ran into was if the image wasn't found, the ALT text caused the column to blow out